File: app/src/main/java/com/example/gymapp/HorarioClass.java
package com.example.gymapp;
import android.os.Bundle;

import java.util.Date;

class HorarioClass {

    private int id;
    private String entrada;
    private String salida;
    private Date fecha;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEntrada() {
        return entrada;
    }

    public void setEntrada(String entrada) {
        this.entrada = entrada;
    }

    public String getSalida() {
        return salida;
    }

    public void setSalida(String salida) {
        this.salida = salida;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }
}

File: app/src/main/java/com/example/gymapp/HorarioClass.kt
package com.example.gymapp

import java.util.Date

class HorarioClass {

    var entrada: String? = null;
    var salida: String? = null;
    var fecha: Date? = null;
    constructor(entrada:String, salida:String, fecha:Date){

        this.entrada = entrada
        this.salida = salida
        this.fecha = fecha
    }


}